### DISCUSSION TITLE
2023-07-19 Data Architecture Meeting

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_mecha#218] [Meeting Agendas]
- https://github.com/SainsburyWellcomeCentre/aeon_mecha/pull/206
- SciViz Updates
  - Video Streamer
- JSON Schema Data Contract (e.g. https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/175)
- Social exp data, database structure, and data ingestion

**Comment by ttngu207** [comment]
We discussed updating the SLEAP loader to be able to handle searching/loading model paths internally. Users do not need to specify model paths.
This requires  more formal model paths organization with some convention, Goncalo will look more into this and come up with a design.
 
 See discussion [here](https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/242)
